pkgs/applications/virtualization/firectl/default.nix

```diff
@@ -2,18 +2,18 @@
 
 buildGoModule rec {
   pname = "firectl";
-  version = "0.1.0";
-
-  patches = [ ./gomod.patch ];
+  # The latest upstream 0.1.0 is incompatible with firecracker
+  # v0.1.0. See issue: https://github.com/firecracker-microvm/firectl/issues/82
+  version = "0.1.0-9f1b639a";
```

Comment:
We generally use 'unstable-<date>' as the revision number here. There's some open discussion on this, but that's still pending so not sure if you should look at that yet (https://github.com/NixOS/rfcs/pull/107).

Reply:
Thank you for the great feedback! I'll update the version number to `unstable-YYYY-MM-DD`, since that looks to be more widely used.